mix.exs

```diff
@@ -61,6 +61,7 @@ defmodule MongoosePush.Mixfile do
       {:telemetry, "~>0.4.1"},
       {:telemetry_metrics, "~> 0.5"},
       {:telemetry_metrics_prometheus_core, "~> 0.4"},
+      {:telemetry_poller, "~> 0.4"},
```

Comment:
Can we use version `0.5`? It seems to be the newest on hex.pm.

Reply:
Will have that updated.